etcdmain/config.go

```diff
@@ -143,6 +144,9 @@ func newConfig() *config {
 	fs.Int64Var(&cfg.QuotaBackendBytes, "quota-backend-bytes", cfg.QuotaBackendBytes, "Raise alarms when backend size exceeds the given quota. 0 means use the default quota.")
 	fs.UintVar(&cfg.MaxTxnOps, "max-txn-ops", cfg.MaxTxnOps, "Maximum number of operations permitted in a transaction.")
 	fs.UintVar(&cfg.MaxRequestBytes, "max-request-bytes", cfg.MaxRequestBytes, "Maximum client request size in bytes the server will accept.")
+	fs.DurationVar(&cfg.KeepAliveEnforcementPolicyMinTime, "keepalive-min-time", time.Duration(0), "Minimum interval duration that a client should wait before pinging server.")
```

Comment:
@gyuho @heyitsanthony 

should we give a good default value? can we add gRPC prefix to these flags?